docs/debugging/s3-check-md5/main.go

```diff
@@ -57,6 +60,7 @@ func main() {
 	flag.BoolVar(&debug, "debug", false, "Prints HTTP network calls to S3 endpoint")
 	flag.BoolVar(&versions, "versions", false, "Verify all versions")
 	flag.BoolVar(&insecure, "insecure", false, "Disable TLS verification")
+	flag.StringVar(&minModTimeStr, "minimum-modtime", "", "Specify a minimum object last modified time, e.g.: 2023-01-02T15:04:05Z")
```

Comment:
Should we name it more nicer saying --modified-since

Reply:
> Should we name it more nicer saying --modified-since

That's too nice for a debugging tool :)

Done